### PR TITLE
Migrate Video Tile storybook to V8

### PR DESCRIPTION
**Pull request by prabhjot-msft**
# What
Migrate Video Tile storybook to V8

Screenshot 2024-07-02 at 5 15 50 PM

Screenshot 2024-07-02 at 5 16 03 PM

Screenshot 2024-07-02 at 5 16 19 PM

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3774818

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook8/stories/constants.ts [hunk 1]
```diff
@@ -59,6 +59,8 @@ export const overviewPageUIArtifactImageStyle = {
   padding: '0rem 1rem 1rem 1rem'
 };
 
+export const COMPONENT_FOLDER_PREFIX = 'UI Components';
```
Comment (AmyL219) [inline packages/storybook8/stories/constants.ts:62]: Where is const being used?

File: packages/storybook8/stories/Components/VideoTile/VideoTile.story.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { StreamMedia, VideoTile as VideoTileComponent } from '@azure/communication-react';
+import React from 'react';
+import { useVideoStreams } from '../../utils';
+
+const VideoTileRender = (args: any): JSX.Element => {
```
Comment (AmyL219) [inline packages/storybook8/stories/Components/VideoTile/VideoTile.story.tsx:7]: where is this VideoTileRender being used?

File: packages/storybook8/stories/Components/VideoTile/snippets/StylingVideoTile.snippet.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  VideoTile,
+  VideoTileStylesProps,
+  DEFAULT_COMPONENT_ICONS
+} from '@azure/communication-react';
+import { initializeIcons, registerIcons, Stack } from '@fluentui/react';
+import React, { useState } from 'react';
+import { renderVideoStream } from '../../../utils';
+
+initializeIcons();
+registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
```
Comment (JoshuaLai) [inline packages/storybook8/stories/Components/VideoTile/snippets/StylingVideoTile.snippet.tsx:15]: As we discussed moving this into the `manager.tsx`

**Review by JoshuaLai — APPROVED** [review]
Looks good just need to resolve the outstanding comments

File: packages/storybook8/stories/Components/VideoTile/snippets/StylingVideoTile.snippet.tsx (hunk 3 above)
Comment (prabhjot-msft) [reply packages/storybook8/stories/Components/VideoTile/snippets/StylingVideoTile.snippet.tsx:15]: Tried moving to `manager.tsx`, it is throwing error, keeping it as it is for now, will keep investigating further.

**Review by AmyL219 — APPROVED** [review]

**Pull request merged**